usvm-jvm/src/main/kotlin/org/usvm/machine/JcApproximations.kt

```diff
@@ -110,66 +85,84 @@ class JcMethodApproximationResolver(
     }
 
     private fun approximateRegularMethod(methodCall: JcMethodCall): Boolean = with(methodCall) {
-        if (method.enclosingClass == usvmApiSymbolicList) {
+        val enclosingClass = method.enclosingClass
+        val className = enclosingClass.name
+
+        if (enclosingClass == usvmApiSymbolicList) {
             approximateUsvmSymbolicListMethod(methodCall)
             return true
         }
 
-        if (method.enclosingClass == usvmApiSymbolicMap) {
+        if (enclosingClass == usvmApiSymbolicMap) {
             approximateUsvmSymbolicMapMethod(methodCall)
             return true
         }
 
-        if (method.enclosingClass == usvmApiSymbolicIdentityMap) {
+        if (enclosingClass == usvmApiSymbolicIdentityMap) {
             approximateUsvmSymbolicIdMapMethod(methodCall)
             return true
         }
 
-        if (method.enclosingClass == ctx.cp.objectClass) {
+        if (enclosingClass == ctx.cp.objectClass) {
             if (approximateObjectVirtualMethod(methodCall)) return true
         }
 
-        if (method.enclosingClass == ctx.classType.jcClass) {
+        if (enclosingClass == ctx.classType.jcClass) {
             if (approximateClassVirtualMethod(methodCall)) return true
         }
 
-        if (method.enclosingClass.name == "jdk.internal.misc.Unsafe") {
+        if (className == "jdk.internal.misc.Unsafe") {
             if (approximateUnsafeVirtualMethod(methodCall)) return true
         }
 
-        if (method.name == "clone" && method.enclosingClass == ctx.cp.objectClass) {
+        if (method.name == "clone" && enclosingClass == ctx.cp.objectClass) {
             if (approximateArrayClone(methodCall)) return true
         }
 
+        if (className == "org.apache.commons.logging.Log") {
```

Comment:
Maybe we can remove non-standard classes from JcApproximations and write this part in `java-stdlib-approximations`?